internal/server/node/property_value.go

```diff
@@ -111,26 +106,32 @@ func GetPropertyValuesHelper(
 	dcids []string,
 	prop string,
 	arcOut bool,
-) (map[string][]*model.Node, error) {
+) (map[string][]*pb.EntityInfo, error) {
 	rowList := bigtable.BuildPropertyValuesKey(dcids, prop, arcOut)
 	baseDataList, _, err := bigtable.Read(
 		ctx,
 		store.BtGroup,
 		rowList,
-		func(dcid string, jsonRaw []byte) (interface{}, error) {
-			var propVals model.PropValueCache
-			if err := json.Unmarshal(jsonRaw, &propVals); err != nil {
+		func(dcid string, jsonRaw []byte, isProto bool) (interface{}, error) {
+			var propVals pb.EntityInfoCollection
+			var err error
+			if isProto {
+				err = proto.Unmarshal(jsonRaw, &propVals)
+			} else {
+				err = protojson.Unmarshal(jsonRaw, &propVals)
+			}
+			if err != nil {
 				return nil, err
 			}
-			return propVals.Nodes, nil
+			return propVals.Entities, nil
```

Comment:
nit:

line 123-126 can be simply:

return propVals.Entities, err

Reply:
Done